Tests/SwiftBasicFormatTest/BasicFormatTests.swift

```diff
@@ -250,6 +250,23 @@ final class BasicFormatTest: XCTestCase {
     )
   }
 
+  func testMultilineStringLiteralWithBlackLines() {
```

Comment:
```suggestion
  func testMultilineStringLiteralWithBlankLines() {
```

I made a typo in the test method name.

Reply:
Haha, thanks for pointing it out.